apps/calendar/js/bridge.js

```diff
@@ -91,10 +94,70 @@ var persistEvent = co.wrap(function *(event, action, capability) {
   return provider[action + 'Event'](event.data);
 });
 
-exports.getSetting = co.wrap(function *(id) {
-  var settingsStore = core.storeFactory.get('Setting');
-  return yield settingsStore.getValue(id);
-});
+exports.getSetting = function(id) {
```

Comment:
we need the default value just for the initial display/instantiation.. we don't need to update it during the lifecycle of the view. so `getSetting` makes sense in this case and is simpler.